### PR TITLE
REG-4 | added random object into the list

### PR DESCRIPTION
**Pull request by nazarrachunn**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,25 +1,43 @@
...
+
+  useEffect(() => {
+    fetch("https://jsonplaceholder.typicode.com/todos")
+      .then((response) => response.json())
+      .then((json) => {
+        JSON.stringify(
+          setRandomItem(json[Math.floor(Math.random() * json.length)])
+        );
+      });
+  }, [randomItem]);
```
Comment (taras-bilozir-globacap) [inline src/App.tsx]: `randomItem` will never be equal new version of `randomItem` since it's an object.

File: src/App.tsx [hunk 2]
```diff
@@ -1,25 +1,43 @@
-import React from "react";
-import logo from "./logo.svg";
+import React, { useEffect, useState } from "react";
 import "./App.css";
 
 function App() {
+  const [items, setItems] = useState([]);
+  const [randomItem, setRandomItem] = useState();
```
Comment (taras-bilozir-globacap) [inline src/App.tsx]: Second `useState` is not required since we no longer render a single item.

File: src/App.tsx [hunk 3]
```diff
@@ -1,25 +1,43 @@
...
+  }, []);
+
+  useEffect(() => {
+    fetch("https://jsonplaceholder.typicode.com/todos")
+      .then((response) => response.json())
+      .then((json) => {
+        JSON.stringify(
+          setRandomItem(json[Math.floor(Math.random() * json.length)])
+        );
+      });
```
Comment (taras-bilozir-globacap) [inline src/App.tsx]: Second `useEffect` is not required since we no longer render a single item.

File: src/App.tsx [hunk 4]
```diff
@@ -1,25 +1,43 @@
...
+  }, [randomItem]);
+
+  const handleGenerateItem = () => {
+    randomItem && setItems([randomItem, ...items]);
```
Comment (taras-bilozir-globacap) [inline src/App.tsx]: `setItems([...items, { id: 100000,  ... }]);`

**Review by taras-bilozir-globacap — CHANGES_REQUESTED** [review]

**Review by taras-bilozir-globacap — CHANGES_REQUESTED** [review]
Requested changes.

File: src/App.tsx (hunk 1 above)
Comment (nazarrachunn) [reply src/App.tsx]: Done

File: src/App.tsx (hunk 2 above)
Comment (nazarrachunn) [reply src/App.tsx]: Done

File: src/App.tsx (hunk 3 above)
Comment (nazarrachunn) [reply src/App.tsx]: Done

File: src/App.tsx (hunk 4 above)
Comment (nazarrachunn) [reply src/App.tsx]: Done

**Review by taras-bilozir-globacap — APPROVED** [review]
Looks good.

**Pull request merged**